authors/apps/comments/tests/test_comments.py

```diff
@@ -0,0 +1,132 @@
+
+    def test_get_all_comments(self):
+        """
+        Test retrieve all comments
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        self.create_comment(
+            self.sample_comment, self.sample_article)
+        response = self.client.get(self.comment_url("my-article"))
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_get_one_comment(self):
+        """
+        Test retrieve one comment
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        res = self.create_comment(
+            self.sample_comment, self.sample_article)
+        url = self.comment_url("my-article")+'5/'
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_update_comment(self):
+        """
+        Test a user can update their comment
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        res = self.create_comment(
+            self.sample_comment, self.sample_article)
+        url = self.comment_url("my-article")+'7/'
+        response = self.client.put(
+            url,
+            self.sample_update_data,
+            format='json'
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            response.data['comment']['message'],
+            "Comment Updated successfully"
+        )
+
+    def test_unauthorized_update_comment(self):
+        """
+        Test a user cannot update other user's comment
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        res = self.create_comment(
+            self.sample_comment, self.sample_article)
+        self.authenticate_user(self.sample_bad_user)
+        url = self.comment_url("my-article")+'6/'
+        response = self.client.put(
+            url,
+            self.sample_update_data,
+            format='json'
+        )
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEqual(response.data['comment']
+                         ['message'], "You are not authorized to comment")
+
+    def test_delete_comment(self):
+        """
+        Test delete comment
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        res = self.create_comment(
+            self.sample_comment, self.sample_article)
+        url = self.comment_url("my-article")+'2/'
+        response = self.client.delete(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['comment']
+                         ['message'], "Comment deleted successfully")
+
+    def test_get_inexistent_comment(self):
```

Comment:
There seems to be some grammatical error here, kindly correct the function name.